AutomatedTesting/Gem/PythonTests/AWS/Windows/aws_metrics/aws_metrics_automation_test.py

```diff
@@ -200,6 +201,59 @@ def test_realtime_and_batch_analytics(self,
         for thread in operational_threads:
             thread.join()
 
+    @pytest.mark.parametrize('level', ['AWS/Metrics'])
+    def test_realtime_and_batch_analytics_no_global_accountid(self,

```

Comment:
Maybe I'm wrong but I think this piece of code for the metrics gem test is already included in another PR https://github.com/o3de/o3de/pull/5475?

Same for the AWSCore test and resource mapping fixture changes.

Reply:
It does contain those changes! It needs to build off of them to work, so they're included as separate commits right now but my hope is they will go away after that PR is merged. I mentioned this in the PR description, but it's easy to miss (I myself tend to head right for the file diff...)